recipes/libignition-sensors/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set component_name = "sensors" %}
+{% set base_name = "libignition-" + component_name %}
+{% set version = "4.1.0" %}
+{% set major_version = version.split('.')[0] %}
+{% set name = base_name + major_version %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ignitionrobotics/ign-{{ component_name }}/archive/ignition-{{ component_name }}{{ major_version }}_{{ version }}.tar.gz
+    sha256: d3a3ae2d635323c0757a6d1bb17c5ccbd990ab491aa5136ad4934dbbeab6bebe
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - make  # [not win]
+    - cmake
+    - pkg-config
+  host:
+    - libignition-cmake2
+    - libprotobuf
+    - libignition-math6
+    - libignition-common3
+    - libignition-transport9
+    - libignition-rendering4
+    - libignition-msgs6
+    - libignition-plugin1
+    - libsdformat10
+  run:
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/ignition/{{ component_name }}{{ major_version }}/ignition/{{ component_name }}.hh  # [not win]
+    - test -f ${PREFIX}/lib/libignition-{{ component_name }}{{ major_version }}.so  # [linux]
+    - test -f ${PREFIX}/lib/libignition-{{ component_name }}{{ major_version }}.dylib  # [osx]
+    - test -f ${PREFIX}/lib/cmake/ignition-{{ component_name }}{{ major_version }}/ignition-{{ component_name }}{{ major_version }}-config.cmake  # [not win]
+    - if exist %PREFIX%\\Library\\include\\ignition\\{{ component_name }}{{ major_version }}\\ignition\\{{ component_name }}.hh (exit 0) else (exit 1)  # [win]
+    - if exist $PREFIX$\\Library\\lib\\ignition-{{ component_name }}{{ major_version }}.lib (exit 0) else (exit 1)  # [win]
+    - if exist $PREFIX$\\Library\\bin\\ignition-{{ component_name }}{{ major_version }}.dll (exit 0) else (exit 1)  # [win]
+    - if exist %PREFIX%\\Library\\lib\\cmake\\ignition-{{ component_name }}{{ major_version }}\\ignition-{{ component_name }}{{ major_version }}-config.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/ignitionrobotics/ign-{{ component_name }}
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: Ignition Robotics library that provide numerous sensor models designed to generate realistic data from simulation environments. 
+
+extra:
```

This is how you get the number out of the feedstock name.
```suggestion
extra:
  feedstock-name: libignition-sensors
```

---

Thanks!